fakturogen-user/src/main/java/pl/fakturogen/user/dao/entity/UserExternalApi.java

```diff
@@ -12,7 +12,7 @@
 @Entity
 @Table(name = UserExternalApi.TABLE_NAME)
 public class UserExternalApi {
-    public static final String TABLE_NAME = "users_external_api";
+    public static final String TABLE_NAME = "users external api";
```

Comment:
I think that we should have previous version "users_external_api" current version doesn't have '_' between words